### DISCUSSION TITLE
Inquiry about Default Query String Parser

### DISCUSSION THREAD

**Discussion by cvbn55688** [fastify/fastify#5281] [Q&A] [answered]
Hi Community,

I have a stupid question.

The documentation mentions that Fastify's default query string parser is Node.js's core querystring module. However, I've been browsing through the source code and couldn't find any direct references to the `node:querystring` module.

In my understanding, there should be a segment of code that handles the scenario where if no `querystringParser` option is provided, it would default to using Node.js's querystring. However, I seem to have missed any related configuration in the source code.

Any help or guidance would be greatly appreciated!

Thanks in advance for your help!

**Comment by metcoder95** [comment]
👋 
The querystring parser is forwarded to `find-my-way`, if not provided, `find-my-way` defaults to `fast-querystring` module:

>The default query string parser that find-my-way uses is [fast-querystring](https://www.npmjs.com/package/fast-querystring) module. You can change this default setting by passing the option querystringParser and use a custom one, such as [qs](https://www.npmjs.com/package/qs).

I believe that we might have a small issue with the documentation.

**Comment by smith558** [comment, marked as answer]
This has now been fixed.
